NOCODE-CHANGELOG.md

```diff
@@ -18,3 +18,4 @@ YouTube (or blog)
 
 | CONTRIBUTOR | CONTRIBUTION | DATE (YYYY/MM/DD) | RESOURCE LINK (PR, Blog, Video) |
 | :---------- | :----------- | :---------------- | :------------------------------ |
+| [Christine Belzie](https://github.com/CBID2/CBID2)|Pair programmed on Discord| 2022/10/29|[#2060](https://github.com/EddieHubCommunity/LinkFree/pull/2060)|
```

Comment:
I think it might be better to add your profile not your repo? https://github.com/CBID2

note: also markdown renders are quite sensitive to spacing, I suggest having a space on either side around `|`